src/core/connection.c

```diff
@@ -4382,6 +4382,8 @@ QuicConnRecvFrames(
             case QUIC_FRAME_ACK_1:
             case QUIC_FRAME_CRYPTO:
             case QUIC_FRAME_CONNECTION_CLOSE:
+            case QUIC_FRAME_STREAMS_BLOCKED:
+            case QUIC_FRAME_STREAMS_BLOCKED_1:
```

Comment:
I don't think these are actually allowed. You should be delaying sending these until 1-RTT.

Reply:
My comment on send.c will fix this.

Reply:
Please remove these lines.

Reply:
Oh, thanks for point it out, I thought I did that, will remove.